.devcontainer/devcontainer.json

```diff
@@ -5,6 +5,7 @@
   "remoteUser": "vscode",
   "workspaceFolder": "/workspaces/${localWorkspaceFolderBasename}",
   "forwardPorts": [3000],
+  "runArgs": ["--network=host"],
```

Comment:
might need to be tested on mac? iirc network=host is different on mac and linux

Reply:
They do not work by default though; it seems this is because Docker host networking is only supported on Linux